k8s_test_harness/util/k8s_util.py

```diff
@@ -114,3 +123,73 @@ def get_join_token(
 # Join an existing cluster.
 def join_cluster(instance: harness.Instance, join_token: str):
     instance.exec(["k8s", "join-cluster", join_token])
+
+
+def get_helm_command(
```

Comment:
It would be great it if this helper could also install helm charts from git repos, some older charts haven't been published on OCI repos.

I had to do it here: https://github.com/canonical/multus-rock/pull/7/files#diff-1c77b059234bd471721a32d318c86695aed1274e8012d3179e1130a5a556b8eeR20-R28.

Reply:
nit: should we rename this to "get_helm_install_command"?

Reply:
Renamed function.

For the git clone part, I think that should be a separate function, and then pass the local folder of the git clone as the chart_name.